src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -98,6 +111,10 @@ public function __construct()
 
         // Add the default, local resource search path:
         $this->searchPaths[] = __DIR__ . "/../Resources";
+        
+        // blacklist php provided auth based values
+        $this->blacklist('_SERVER', 'PHP_AUTH_USER');
```

Comment:
here you can see the public api a whoops user would call for this feature in action.